style.css

```diff
@@ -0,0 +1,417 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  margin: 0 auto;
+  border: 1px solid black;
```

Comment:
- [ ] kindly remove the border around the portfolio so as to follow the original design